docs/how-to-guides/test-an-app.md

```diff
@@ -8,161 +8,168 @@ keywords: testing, qa, quality assurance, test runner
 
 # How to test an application
 
+!!! important
+    ksqlDB 0.30 ships with a new sql-based testing tool. For documentation on the old test runner,
+    switch to an older version of this page.
+
 ## Context
 
-You have an application of ksqlDB statements, and you want to automatically test whether they behave correctly when given a set of inputs. ksqlDB exposes a *test runner* command line tool to do just that. It runs quickly and doesn't require a running {{ site.aktm }} or ksqlDB cluster.
+ksqlDB ships with a command line tool to to test KSQL statements automatically.
```

Comment:
It is the second change and no name of the tool yet? How about adding `run-ksql-test` to the `important` section above (since it will surely draw people attention to a lot) or here in Context?